### PR TITLE
chore: fix nightly jobs and allow choosing flows

### PR DESCRIPTION
**Pull request by radoslaw-szpiech**
# Related issues 🔗

Closes #[Issue number here]

# Description ℹ️

A description of what this PR does. This should be non-dev readable. Technical stuff should go in the technical section below.

# Demo 📺

Gif/video/images of new/corrected functionality if applicable

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: .github/workflows/capsule-cypress-nightly.yml [hunk 1]
```diff
@@ -149,7 +107,7 @@ jobs:
           CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE: ${{ secrets.CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE }}
           CYPRESS_SLACK_WEBHOOK: ${{ secrets.CYPRESS_SLACK_WEBHOOK }}
           CYPRESS_ETH_WALLET_MNEMONIC: ${{ secrets.CYPESS_ETH_WALLET_MNEMONIC }}
-          CYPRESS_NIGHTLY_RUN: true
+          CYPRESS_INCLUDE_FLOWS: true
```
[the remark this answers is not in the document]
Comment (radoslaw-szpiech) [reply .github/workflows/capsule-cypress-nightly.yml:110]: That's what I thought, but: 
image

[the remark this answers is not in the document]
Comment (jtsang586) [reply .github/workflows/capsule-cypress-nightly.yml:110]: Remove the Cypress `NIGHTLY_RUN` 😄

[the remark this answers is not in the document]
Comment (radoslaw-szpiech) [reply .github/workflows/capsule-cypress-nightly.yml:110]: ... Changing it right now.... xD

File: .github/workflows/capsule-cypress-manual-trigger.yml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        type: boolean
+        default: true
+
+jobs:
+  manual:
+    name: Run capsule tests -- manual trigger
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: 'on'
+      GOPROXY: ${{ secrets.GO_PROXY }}
```
Comment (mkjmdski) [inline .github/workflows/capsule-cypress-manual-trigger.yml:30]: You don't need that anymore

File: .github/workflows/capsule-cypress-manual-trigger.yml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        run: vegacapsule install-bins
+        env:
+          GITHUB_TOKEN: ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }}
+          GOBIN: ${{ env.GOBIN }}
+
+      ######
+      ## Start capsule
+      ######
+      - name: Login to docker
+        run: echo -n ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }} | docker login https://ghcr.io -u vega-ci-bot --password-stdin
```
Comment (dexturr) [inline .github/workflows/capsule-cypress-manual-trigger.yml:79]: chore (non-blocking): we can kill this now the containers are public

File: .github/workflows/capsule-cypress-manual-trigger.yml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+      - name: Install root dependencies
+        run: yarn install
+
+      - name: Run Cypress tests
+        run: yarn nx run-many --target=e2e --project=${{inputs.project}} --record --key ${{ secrets.CYPRESS_RECORD_KEY }} --browser chrome
+        env:
+          CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE: ${{ secrets.CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE }}
+          CYPRESS_SLACK_WEBHOOK: ${{ secrets.CYPRESS_SLACK_WEBHOOK }}
+          CYPRESS_ETH_WALLET_MNEMONIC: ${{ secrets.CYPESS_ETH_WALLET_MNEMONIC }}
```
[the remark this answers is not in the document]
Comment (radoslaw-szpiech) [reply .github/workflows/capsule-cypress-manual-trigger.yml:124]: https://github.com/vegaprotocol/frontend-monorepo/issues/781

File: .github/workflows/capsule-cypress-manual-trigger.yml [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+
+      - name: Install root dependencies
+        run: yarn install
+
+      - name: Run Cypress tests
+        run: yarn nx run-many --target=e2e --project=${{inputs.project}} --record --key ${{ secrets.CYPRESS_RECORD_KEY }} --browser chrome
+        env:
+          CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE: ${{ secrets.CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE }}
```
[the remark this answers is not in the document]
Comment (radoslaw-szpiech) [reply .github/workflows/capsule-cypress-manual-trigger.yml:122]: https://github.com/vegaprotocol/frontend-monorepo/issues/781

File: .github/workflows/capsule-cypress-manual-trigger.yml (hunk 3 above)
Comment (radoslaw-szpiech) [reply .github/workflows/capsule-cypress-manual-trigger.yml:79]: https://github.com/vegaprotocol/frontend-monorepo/issues/781

**Review by jtsang586 — APPROVED** [review]
Lgtm

**Review by AndyWhiteVega — APPROVED** [review]

**Pull request merged**